packages/admin/cms-admin/src/preview/SitePreview.tsx

```diff
@@ -21,35 +21,41 @@ interface SiteState {
     includeInvisibleBlocks: boolean;
 }
 
-interface Props extends RouteComponentProps {
+interface Props {
```

Comment:
Is this a breaking change? Won't TS complain about passing too many props to the SitePreview component?

Reply:
yes, sorry.